cli/cmd/stat_summary.go

```diff
@@ -146,19 +150,21 @@ func writeStatTableToBuffer(resp *pb.StatSummaryResponse, w *tabwriter.Writer) {
 		"IN_RPS",
 		"IN_SUCCESS",
 		"IN_LATENCY_P50",
+		"IN_LATENCY_P95",
```

Comment:
Ah, this makes me realize that the column headers we're using here aren't correct. Per the updates to #627, the column headers should be:

```
NAME           MESHED      SUCCESS      RPS      LATENCY_P50      LATENCY_P95
```

Would you mind making that change as part of this branch?